.gitattributes

```diff
@@ -0,0 +1,63 @@
+# Code
+*.py text eol=lf diff=python
```

Comment:
permit to have better diff for python if version of git contain the diff parser for python.